tsconfig.json

```diff
@@ -3,6 +3,8 @@
     "target": "esnext",
     "module": "esnext",
     "strict": false,
+    "noImplicitAny": false,
+    "strictNullChecks": false,
```

Comment:
Added the two rules to stay consistent between all the 3 UIs.

Reply:
Might want to add in https://www.typescriptlang.org/tsconfig/skipLibCheck.html

What do you think Severin?

Reply:
Do we skip type checking of declaration files anywhere else? I'd rather my build take a few extra seconds than give up type-checking :)

Reply:
Maybe not such a good idea considering `bcrs-shared-components` isn't precompiled

Reply:
Hahaha